### DISCUSSION TITLE
How do you change the various font sizes in react?

### DISCUSSION THREAD

**Discussion by aboyher** [upsetjs/upsetjs#81] [Q&A] [answered]
The set labels on my plots tend to overflow and I would like to decrease the font size. How do I go about doing that in react.

My typically call looks like this:

```
<UpSetJS
     sets={sets}
     combinations={combinations}
     selection={this.state.selection}
     onHover={this.setSelection} 
     width={width}
     height={height}
     title={title}
     theme={theme}
/>
```

**Comment by sgratzl** [comment, marked as answer]
take a look at https://upset.js.org/api/react/interfaces/upsetpropsg.html#fontsizes and https://upset.js.org/api/react/interfaces/upsetpropsg.html#setlabelalignment

**Reply by aboyher** [reply]
Hey Thanks! I finally figured it out. 

Per the license, am I allowed to publish this to pypi? I find it immensely useful and I think others would find it useful as well.
